spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/segmentation/Resize.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.dataset.segmentation
+
+import com.intel.analytics.bigdl.transform.vision.image.{FeatureTransformer, ImageFeature}
+import com.intel.analytics.bigdl.transform.vision.image.label.roi.RoiLabel
+import com.intel.analytics.bigdl.transform.vision.image.util.BboxUtil
+import org.opencv.core.Size
+import org.opencv.imgproc.Imgproc
+
+class Resize(minSize: Int, maxSize: Int = -1, resizeROI: Boolean = false)
```

Comment:
Can we name it a different one and put it under the same package as other transformer ?

Reply:
ok, changed